File: 03-03-2026/flattened.py
def is_flat(arr):
    for item in arr:
        if isinstance(item, list):
            return False
    return True

print(is_flat([1, 2, 3, 4])); # true
print(is_flat([1, [2, 3], 4])); # false
print(is_flat([1, 2, [3, 4], 5])); # false
print(is_flat([1, 2, 3, 4, 5])); # true

File: 03-03-2026/flattened.js
function isFlat(arr) {
  return arr.every((item) => !Array.isArray(item));
}

console.log(isFlat([1, 2, 3, 4])); // true
console.log(isFlat([1, [2, 3], 4])); // false
console.log(
  isFlat([
    [1, 2],
    [3, 4],
  ]),
); // false
console.log(isFlat([1, 2, [3, 4], 5])); // false
console.log(isFlat([1, 2, 3, 4, 5])); // true
